NOTICE.md

```diff
@@ -0,0 +1,47 @@
+# Notices for Eclipse Tractus-X
+
+This content is produced and maintained by the Eclipse Tractus-X project.
+
+* Project home: <https://projects.eclipse.org/projects/automotive.tractusx>
+
+See the AUTHORS file(s) distributed with this work for additional information regarding authorship.
+
+## Trademarks
+
+Eclipse Tractus-X is a trademark of the Eclipse Foundation.
+
+## Copyright
+
+All content is the property of the respective authors or their employers. For
+more information regarding authorship of content, please consult the listed
+source code repository logs.
+
+## Declared Project Licenses
+
+This program and the accompanying materials are made available under the terms
+of the Apache License, Version 2.0 which is available at
+<https://www.apache.org/licenses/LICENSE-2.0>.
+
+SPDX-License-Identifier: Apache-2.0
+
+## Source Code
+
+The project maintains the following source code repositories
+in the GitHub organization <https://github.com/eclipse-tractusx>:
+
+* <https://github.com/eclipse-tractusx/tractusx-edc>
```

Comment:
this is the wrong repo...

Reply:
Addressed with: https://github.com/eclipse-tractusx/ssi-dim-wallet-stub/commit/33b9f3814158b68e664ee0efae999956111fe5b2